src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -464,9 +464,11 @@ GetShellTableDDLEventsForCitusLocalTable(Oid relationId)
 	 * a sequence.
 	 */
 	bool includeSequenceDefaults = true;
+	bool nextvalInWorkerNode = false;
 
 	List *tableDDLCommands = GetFullTableCreationCommands(relationId,
-														  includeSequenceDefaults);
+														  includeSequenceDefaults,
+														  nextvalInWorkerNode);
```

Comment:
could this be an enum that encapsulates the 3 values?
- no default
- nextval as default
- worker_nextval as default